pkg/operator/ceph/pool/validate.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2020 The Rook Authors. All rights reserved.
```

The validation could go into `pkg/operator/ceph/object/objectstore.go` as well.

Reply:
> The validation could go into pkg/operator/ceph/object/objectstore.go as well.

The object store already calls these validation methods. Is that what you mean?